ext/rugged/rugged.h

```diff
@@ -147,6 +147,12 @@ struct rugged_cb_payload
     int exception;
 };
 
+struct rugged_treecount_cb_payload
```

Comment:
Absolutely. I actually moved it in here as that's where it looked like all of the other payloads were defined. I'll move it back.